flipkart-guardrails/src/main/java/com/flipkart/fdp/AdjustJobConfigurationImpl.java

```diff
@@ -125,6 +129,11 @@ public void adjustJobConfiguration(Configuration jobConf) {
     }
 
     ensureAllPropertiesAreAvailable(processDataProperties);
+    try{
+      checkInputFilesSize(jobConf);
+    }catch (IOException io){
+      new RuntimeException("IO exception occured while checking input paths");
```

Comment:
This is the IO exception not exception of FileSystem.getContentSummary method. For input file size exception it has been mentioned.